CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 - [835](https://github.com/umee-network/umee/pull/835) Add miss counter query to oracle cli.
+- [860](https://github.com/umee-network/umee/pull/860) Add IBC upgrade and upgrade-client gov proposals
```

Comment:
```suggestion
```

Please move this under the ## [Unreleased] section 😄 details for our cl process are in this file